SUIApp/SUIApp/ViewModel/ProductViewModel.swift

```diff
@@ -0,0 +1,80 @@
+// ProductViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// ViewModel для товаров
+final class ProductViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let sofaTitleText = "Sofa"
+        static let armchairTitleText = "Armchair"
+        static let bedTitleText = "Bed"
+        static let chair = "Chair"
+        static let oldPriceText = "79"
+        static let emptyText = ""
+        static let zeroNumber = 0
+        static let oneNumber = 1
+        static let sofaPriceNumber = 505
+        static let armchairPriceNumber = 220
+        static let bedPriceNumber = 760
+        static let chairPriceNumber = 45
+    }
+
+    // MARK: - Public Property
+
+    @Published var products = [
+        Product(
+            name: Constants.sofaTitleText,
+            imageName: Constants.sofaTitleText.lowercased(),
+            price: Constants.sofaPriceNumber,
+            oldPrice: Constants.oldPriceText,
+            count: Constants.zeroNumber
+        ),
+        Product(
+            name: Constants.armchairTitleText,
+            imageName: Constants.armchairTitleText.lowercased(),
+            price: Constants.armchairPriceNumber,
+            oldPrice: Constants.oldPriceText,
+            count: Constants.zeroNumber
+        ),
+        Product(
+            name: Constants.bedTitleText,
+            imageName: Constants.bedTitleText.lowercased(),
+            price: Constants.bedPriceNumber,
+            oldPrice: Constants.oldPriceText,
+            count: Constants.zeroNumber
+        ),
+        Product(
+            name: Constants.chair,
+            imageName: Constants.chair.lowercased(),
+            price: Constants.chairPriceNumber,
+            oldPrice: Constants.oldPriceText,
+            count: Constants.zeroNumber
+        )
+    ]
+    
+    @Published var searchBarText = Constants.emptyText
+
+    // MARK: - Public Methods
+
+    func fetchProduct(index: Int) -> Product {
+        products[index]
+    }
+
+    func incrementProductCount(value: Int) {
+        products[value].count += Constants.oneNumber
```

Comment:
```suggestion
        products[value].count += 1
```
Отменили такие константы